kbfstool/md_main.go

```diff
@@ -12,6 +12,7 @@ const mdUsageStr = `Usage:
 
 The possible subcommands are:
   dump		Dump metadata objects
+  check		Check metadata objects and their associated blocks for errors
```

Comment:
This looks like it's only downloading things, not really checking their true validity.  Especially for the MD object.  Should we call `BareRootMetadata.IsValidAndSigned()` in `mdGet`?